src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/ResetTask_2.sql

```diff
@@ -0,0 +1,59 @@
+﻿
+/*************************************************************
+    Stored procedures for reset task
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     ResetTask
+--
+-- DESCRIPTION
+--     Reset the task status.
+--
+-- PARAMETERS
+--     @taskId
+--         * The ID of the task record
+--     @runId
+--         * Current runId for this exuction of the task
+--     @result
+--         * The result of the task
+--
+GO
+CREATE PROCEDURE dbo.ResetTask_2
+@taskId VARCHAR (64), @runId VARCHAR (50), @result VARCHAR (MAX)
+AS
+SET NOCOUNT ON;
+SET XACT_ABORT ON;
+DECLARE @retryCount AS SMALLINT = NULL;
+BEGIN TRANSACTION;
```

Comment:
Remove explicit trandaction

Reply:
Could you explain more about it? I'm not very clear the benefit here

Reply:
There is no benefit in having a transaction as it is written. Leaving it makes false impression that it changes behavior in some way, which is not true.

Reply:
it make sense, remove transaction